global-script.lua

```diff
@@ -138,6 +138,23 @@ readyTokens = {
     Green = "88a4c3",
     Orange = "d17f93",
 }
+playerTables = {
```

Comment:
with these now existing in global script state, are these instantiations necessary?

Reply:
I don't like the idea of global script state serving as a source of truth, but that's just my opinion.  It's not something easily discoverable to someone else who may be editing this for the first time.  Plus, the entire script-state.json can be regenerated by deleting it, repacking, and re-saving from inside TTS -- which is exactly what I did.  Removing this from Global would make that no longer possible.

(IMHO, the better option would be to fix whatever makes the setup UI not be visible on first load and ship with a nonexistent script-state.json, but perfect is the enemy of good enough, etc etc.)

Reply:
hmm alright then, currently i got all of the other script state config values in a section. could you move these values to live alongside the others?